tests/src/test/scala/com/nvidia/spark/rapids/HashSortOptimizeSuite.scala

```diff
@@ -106,7 +113,12 @@ class HashSortOptimizeSuite extends FunSuite {
       val df2 = buildDataFrame2(spark)
       val rdf = df1.join(df2, df1("a") === df2("x")).orderBy(df1("a"))
       val plan = rdf.queryExecution.executedPlan
-      val numSorts = plan.map {
+      val finalPlan = plan match {
```

Comment:
It is probably worth putting this logic into a helper method in TestUtils as well? I can imagine we will need to do this in multiple places.

Reply:
moved it to TestUtils